### PR TITLE
✨ Add Home page

### PR DESCRIPTION
**Pull request by J-Bannach**
Home page consists of:
- overview of days 
- meals per day
- select function for meals (functionality not yet implemented)

### CODE REVIEW

File: src/app/components/DaysAndMeals/DaysAndMeals.stories.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import DaysAndMeals from './DaysAndMeals';
 
 export default {
   title: 'Component/Days',
-  component: Text,
+  component: 'DaysAndMeals',
```
Comment (MaxMuecket) [inline src/app/components/DaysAndMeals/DaysAndMeals.stories.tsx:6]:
```suggestion
  component: DaysAndMeals,
```

File: src/app/components/SelectionBlock/SelectionBlock.stories.tsx [hunk 2]
```diff
@@ -1,40 +1,9 @@
 import React from 'react';
-import MealSelection from '../MealSelection/MealSelection';
-import styles from './SelectionBlock.module.css';
+import SelectionBlock from './SelectionBlock';
 
 export default {
   title: 'Component/Link',
   component: 'SelectionBlock',
```
Comment (MaxMuecket) [inline src/app/components/SelectionBlock/SelectionBlock.stories.tsx:6]:
```suggestion
  component: SelectionBlock,
```

**Review by MaxMuecket — APPROVED** [review]
lgtm ✌️

I'm not sure, but maybe this is the reason why your App isn't deploying.

File: src/app/components/SelectionBlock/SelectionBlock.tsx [hunk 3]
```diff
@@ -8,6 +8,30 @@ function SelectionBlock(): JSX.Element {
       <MealSelection link="./search" />
       <MealSelection link="./search" />
       <MealSelection link="./search" />
+      <br />
```
Comment (lmachens) [inline src/app/components/SelectionBlock/SelectionBlock.tsx:11]: You use `br` to create a gap, but this is not how a `br` should be used for:

> Note: Do not use `<br>` to create margins between paragraphs; wrap them in `<p>` elements and use the CSS margin property to control their size.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br

Alternatives:
1) Create blocks
```tsx
  <div>
    <MealSelection link="./search" />
    <MealSelection link="./search" />
    <MealSelection link="./search" />
  </div>
  <div>
    <MealSelection link="./search" />
    <MealSelection link="./search" />
    <MealSelection link="./search" />
  </div>
```
You can style the gap by setting `margin` on the new `div` elements or `row-gap` on the parent grid.

2) Use `:nth-child` CSS selector

```css
.selectionBlock:nth-child(3n) {
  margin-bottom: 1em;
}
```
See https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child

**Review by lmachens — CHANGES_REQUESTED** [review]
gj
Changes are requested because of failing deployment.

File: src/app/components/DaysAndMeals/DaysAndMeals.stories.tsx (hunk 1 above)
Comment (J-Bannach) [reply src/app/components/DaysAndMeals/DaysAndMeals.stories.tsx]: Changed it, thanks.

File: src/app/components/SelectionBlock/SelectionBlock.tsx (hunk 3 above)
Comment (J-Bannach) [reply src/app/components/SelectionBlock/SelectionBlock.tsx]: Thanks, I changed it now to div elements.

**Pull request merged**